src/routes/RemediationDetailsComponents/ActionsContent/ActionsContent.js

```diff
@@ -11,18 +11,20 @@ import ConfirmationDialog from '../../../components/ConfirmationDialog';
 import { actionNameFilter } from '../Filters';
 import SystemsModal from './SystemsModal/SystemsModal';
 import {
-  useRawTableState,
+  useTableState,
   TableStateProvider,
-  StaticTableToolsTable,
   useStateCallbacks,
+  useSerialisedTableState,
 } from 'bastilian-tabletools';
 import TableEmptyState from '../../OverViewPage/TableEmptyState';
 
 import { deleteIssues } from '../../api';
+import RemediationsTable from '../../../components/RemediationsTable/RemediationsTable';
 
 const ActionsContent = ({ remediationDetails, refetch, loading }) => {
   const { id } = useParams();
-  const tableState = useRawTableState();
+  const tableState = useTableState();
```

Comment:
That is not the right hook.

Reply:
@hugohezel This was the reason the bulk remove wasnt working, I wasnt actually accessing the selected values from the table.